services/QuillLMS/config/database.yml

```diff
@@ -27,13 +27,15 @@ heroku_env: &heroku_env
 
 development:
   primary:
-    <<: *local_shared
-    database: emp_gr_development
+    # <<: *local_shared
```

Comment:
just curious -- what's the reasoning behind this change?

Reply:
This is what I do when I want to use the staging DB instead of my local one (you have to set the `DATABASE_URL` in your .env file) but it wasn't supposed to be committed!